test/features/resource.editor.feature

```diff
@@ -92,7 +92,7 @@ Feature: Resource
     When I click "Manage Datastore"
     Then I should see "There is nothing to manage! You need to upload or link to a file in order to use the datastore."
 
-  @resource_editor_6 @datastore @noworkflow @javascript
+  @resource_editor_6 @datastore @noworkflow @javascript @fixme
```

Comment:
Is this related to the PR/bug or just a test that is randomly failing?

Reply:
Not related to the PR, just a redundant test that consistently fails due to the 30 second time limit.